apps/emqx_dashboard/src/emqx_dashboard_schema.erl

```diff
@@ -25,8 +25,18 @@ namespace() -> <<"dashboard">>.
 roots() -> ["dashboard"].
 
 fields("dashboard") ->
-    [ {listeners, hoconsc:array(hoconsc:union([hoconsc:ref(?MODULE, "http"),
-                                               hoconsc:ref(?MODULE, "https")]))}
+    [ {listeners,
+        sc(hoconsc:array(hoconsc:union([hoconsc:ref(?MODULE, "http"),
+            hoconsc:ref(?MODULE, "https")])),
+            #{ desc =>
+"""HTTP(s) listeners identified by their protocol type,
```

Comment:
nit: Add comma after 'listeners' and change 'is' in the next line to 'are' so it reads: HTTP(s) listeners, identified by their protocol type, are used...

Alternatively could rephrase slightly to be: HTTP(s) listeners are identified by their protocol type and are used...